sqldelight-gradle-plugin/src/test/fixtures/works-for-kotlin/src/main/sqldelight/com/test/User.sq

```diff
@@ -4,7 +4,7 @@ CREATE TABLE users (
   middle_initial TEXT,
   last_name TEXT NOT NULL,
   age INTEGER AS Integer NOT NULL DEFAULT 0,
-  gender TEXT AS 'com.test.User.Gender' NOT NULL
+  gender TEXT AS com.test.User.Gender NOT NULL
```

Comment:
@JakeWharton check this out the current grammar handles the inner class/enum stuff i think?